### PR TITLE
Feature/service menu

### PR DESCRIPTION
**Pull request by vikram2141**
dropdown menu change multilevel dropdown @jagadishsail

### CODE REVIEW

File: client/src/components/MenuItems/MenuItems.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    <li>
+                        <Link to="/nutriapp" className="dropdown-item">
+                            Nutrifit
+                        </Link>
+                    </li>
+                   
+                </ul>
+            </li>
+            <li>
+                <Link to="/nutriapp">Nutritional</Link>
```
Comment (geneus-solutions) [inline client/src/components/MenuItems/MenuItems.jsx:34]: Why this Menu Nutritional extra? As discussed this should be part of Services

File: client/src/components/Nutrifit/Nutrifit.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import NutriFitImage from '../../assets/nutrifit.jpeg';
```
Comment (geneus-solutions) [inline client/src/components/Nutrifit/Nutrifit.jsx:2]: Why this file added NutriFit?

File: client/src/components/Services/Services.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Services() {
+  return (
+    <div>Services</div>
+  )
```
Comment (geneus-solutions) [inline client/src/components/Services/Services.jsx:6]: Why this file Services.jsx?

File: client/src/menuItemsData.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+export const menuItems = [
+    {
```
Comment (geneus-solutions) [inline client/src/menuItemsData.js:2]: in which component this file is used? client/src/menuItemsData.js

**Review by geneus-solutions — CHANGES_REQUESTED** [review]

**Comment by vikram2141** [comment]
Update

**Pull request closed without merging**